test/v1/models/request/PriorResidencyModelSpec.scala

```diff
@@ -21,10 +21,14 @@ import support.UnitSpec
 
 class PriorResidencyModelSpec extends UnitSpec {
   val minJson: JsObject = Json.obj()
-  val maxJson: Boolean => JsObject = isWrite => Json.obj(
-    "priorStartDate" -> (if(isWrite) "2020-10-10" else "10-10-2020"),
-    "priorEndDate" -> (if(isWrite) "2020-10-10" else "10-10-2020")
-  )
+  val maxJson: Boolean => JsObject = isWrite => {
+    val startDatePath = if(isWrite) "startDate" else "priorStartDate"
```

Comment:
This could be rewritten as two separate Json.obj. One for write and one for read. Would remove a lot of the if the statements and make it a little bit cleaner.

Reply:
Good shout - done